app/controllers/concerns/umich/catalog.rb

```diff
@@ -0,0 +1,11 @@
+module Umich
+  module Catalog
+    extend ActiveSupport::Concern
+
+    included do
+      Blacklight::Configuration.define_field_access :biography_history_field, Blacklight::Configuration::ShowField
```

Comment:
I believe you can also do this by adding the new field symbols to `config/initializers/arclight.rb`

Reply:
Yes, you are correct, I'll delete it from the PR.  Thanks.